keyboards/keycapsss/plaid_pad/keymaps/via/rules.mk

```diff
@@ -1,2 +1,5 @@
 VIA_ENABLE = yes
 LTO_ENABLE = yes
+VIAL_ENABLE = yes
+VIAL_ENCODERS_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
Can we keep this? It has no negative effect.
Then it easier to build firmware on the vial fork.

Reply:
If VIAL is upstreamed to QMK master we may consider it, but at this stage, no.

Reply:
Done removed it.